.github/workflows/lambda.yml

```
@@ -0,0 +1,47 @@
+name: Lambda
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+  schedule:
+    - cron: '0 4 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/node/setup
+      - run: yarn install
+      - run: yarn test:lambda
```

Should this match the other platforms?

```suggestion
      - run: yarn test:lambda:ci
```